pnpm-lock.yaml

```diff
@@ -95,22 +95,22 @@ importers:
       in-publish: 2.0.1
       jest: 27.0.4
       jest-environment-jsdom: 27.0.3
-      jest-environment-jsdom-global: ^2.0.4
+      jest-environment-jsdom-global: 2.0.4
       jest-environment-node: 27.0.3
       jest-fetch-mock: 3.0.3
       jest-junit: 11.0.1
-      kleur: 4.1.4
-      lint-staged: 8.2.1
+      kleur: 3.0.3
```

Comment:
` kleur: 4.1.4` instead ` kleur: 3.0.3` unless there is a reason.

Reply:
Another bummer. Aligning all kleur to 4.1.4 and tests break. Using v3.0.3 which was last in v3 series and all tests pass.

Reply:
Ok, no problem, let's keep it like this, it is just colors anyway.